commands/util.go

```diff
@@ -280,14 +280,22 @@ func printFullActivationList(activations []whisk.Activation) {
 	}
 }
 
-func printActivationLogs(logs []string) {
+func printStrippedActivationLogs(logs []string) {
 	for _, log := range logs {
-		if Flags.activation.strip {
-			fmt.Printf("%s\n", log[39:])
+		regex := regexp.MustCompile("[a-zA-Z0-9\\s]*(stdout|stderr):\\s(.*)")
+		match := regex.FindStringSubmatch(log)
+
+		if len(match) > 2 && len(match[2]) > 0 {
+			fmt.Printf("%s\n", match[2])
```

Comment:
what do you think about using a utility function for the individual log strip logic
```
func stripTimestamp(log string) strippedLog string {
    regex := regexp.MustCompile("[a-zA-Z0-9\\s]*(stdout|stderr):\\s(.*)")
    match := regex.FindStringSubmatch(log)
    if len(match) > 2 && len(match[2]) > 0 {
        strippedLog = match[2]
    } else {
        strippedLog = log
    }
    return strippedLog
}
```
then the print function would become
```
func printStrippedActivationLogs(logs []string) {
	for _, log := range logs {
		fmt.Printf("%s\n", stripTimestamp(log))
 	}
 }
```
the unit test would just validate the utility function.